Animal.java

```diff
@@ -1,8 +1,15 @@
 public class Animal {
 
-    public String name;
+    public String nname;
     public int age;
     public double weight;
     public double height;
 
+    public void feedAnimal(){
+        System.out.println("Feeding...");
+    }
+
+    public void GoForAWalk(){
```

Comment:
Please check this method name (according to code convension)

Reply:
fixed